dev/drivers/scripts/plots/hurricane/jevs_hurricane_global_ens_spread_plots.sh

```diff
@@ -42,6 +42,9 @@ export PDY=20231231
 #Define the directory for TC-stats file 
 export COMINstats=/lfs/h2/emc/vpppg/noscrub/$USER/evs/${evs_ver_2d}/stats/${COMPONENT}/${RUN}/${VERIF_CASE}
 
+#Define the directories of your NOAA/NWS logos
+export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix
```

Comment:
I had thought about it. Technically it doesn't need to be defined since FIXevs gets defined in the J-Job as $HOMEevs/fix, and that is usually just a soft link to /lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix. Here is just pointing directly to it. If this ran in ops, I would say it should be removed but since this doesn't I'm fine with keeping it or not.

Reply:
Let's go ahead and get rid of `export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix`

Reply:
Can we remove `##PBS -l place=vscatter:exclhost,select=1:ncpus=128:ompthreads=1` to avoid confusion on resources being used?